src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -875,6 +887,21 @@ internal bool IsKeyboardFocusable()
             return ret;
         }
 
+        internal void SetTouchFocusable(bool focusable)
```

Comment:
How about change name for the consistency like below
```suggestion
        internal void SetFocusInTouchMode(bool enabled)
```